README.md

```diff
@@ -450,6 +448,51 @@ If you need to cap the line width, o-typography provides a function which limits
 	max-width: oTypographyMaxLineWidth($scale: 1);
 ```
 
+#### Use A Custom Font
```

Comment:
For the README change, it may be worth mentioning what the default typefaces are (if we don't already)